app/models/concerns/rbac.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+# Role-based Access Control
+module Rbac
+  extend ActiveSupport::Concern
+
+  attr_accessor :loaded_permission
```

Comment:
I suspect there are more than one, so:
```suggestion
  attr_accessor :loaded_permissions
```
Or if this is a boolean, semantically it's more correct to have:
```suggestion
  attr_accessor :permissions_loaded?
```

Reply:
Not boolean, will use `:loaded_permissions`. Thanks! :+1: